scripts/cdn/branch/publish.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Publish packages to our CDN
+ */
+
+// Load secrets file
+
+let fs = require('fs');
+
+let secrets = JSON.parse(fs.readFileSync('secrets.json'));
+if (!secrets ||
+    !secrets.AWS_USER_S3_KEY ||
+    !secrets.AWS_USER_S3_SECRET ||
+    !secrets.AWS_S3_BUCKET) {
+    throw Error('secrets.json content is not valid');
+}
+
+const branchName = require('current-git-branch');
+const branch = JSON.parse(fs.readFileSync('package.json')).branch || branchName();
```

Comment:
I don't understand `JSON.parse(fs.readFileSync('package.json')).branch`, we never define the branch there, right?

Reply:
Nope, we don't. That's why I added a package to get the current branch. The purpose of this is to allow us to use the branch name we want by adding it in the package json, if you prefer I can add it as a script parameter/option instead.

Reply:
To be honest, I wouldn't bother in supporting it in any way